### PR TITLE
Merge 7.0.1 change log

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

Merge 7.0.1 change log

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.

### CODE REVIEW

File: CHANGELOG/7.0.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+
+- Discover assemblies loaded by `Assembly.Load(byte[])` and `Assembly.LoadFile` (#12203)
+- Allow case insensitive paths for determining `PSModulePath` (#12192)
+
+### General Cmdlet Updates and Fixes
+
+- Add `null` check for Windows PowerShell install path (#12296)
+- Fix Null Reference error in CSV commands (#12281) (Thanks @iSazonov!)
+- Fix `WinCompat` module loading to treat Core edition modules higher priority (#12269)
+- Fix `<img />` detection regex in web cmdlets (#12099) (Thanks @!)
```
Comment (daxian-dbw) [inline CHANGELOG/7.0.md:15]: > Thanks @!

File: CHANGELOG/7.0.md [hunk 2]
```diff
@@ -958,6 +1000,7 @@
 - Update release template (#8996)
 
 [7.0.0]: https://github.com/PowerShell/PowerShell/compare/v7.0.0-rc.3...v7.0.0
+[7.0.1]: https://github.com/PowerShell/PowerShell/compare/v7.0.0...v7.0.1
```
Comment (daxian-dbw) [inline CHANGELOG/7.0.md:1003]: Maybe this should be moved above [7.0.0]

File: CHANGELOG/7.0.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
 # 7.0 Changelog
 
+## [7.0.1] - 2020-05-14
```
Comment (daxian-dbw) [inline CHANGELOG/7.0.md:3]: Are we targeting tomorrow for publishing the release? If so, this should be changed to 05-15 I guess

File: CHANGELOG/7.0.md (hunk 1 above)
Comment (TravisEz13) [reply CHANGELOG/7.0.md:15]:
```suggestion
- Fix `<img />` detection regex in web cmdlets (#12099) (Thanks @vex32!)
```

File: CHANGELOG/7.0.md (hunk 3 above)
Comment (TravisEz13) [reply CHANGELOG/7.0.md:3]: It's releasing now.   This is not a review... Just a merge...

File: CHANGELOG/7.0.md (hunk 2 above)
Comment (TravisEz13) [inline CHANGELOG/7.0.md:1003]:
```suggestion
[7.0.1]: https://github.com/PowerShell/PowerShell/compare/v7.0.0...v7.0.1
[7.0.0]: https://github.com/PowerShell/PowerShell/compare/v7.0.0-rc.3...v7.0.0
```

**Comment by daxian-dbw** [comment]
@TravisEz13 please complete the checklist in the PR description.

**Comment by TravisEz13** [comment]
This is not really a PR... It's not applicable.

**Comment by daxian-dbw** [comment]
I was just following the routine 😃

File: CHANGELOG/7.0.md [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+
+- Discover assemblies loaded by `Assembly.Load(byte[])` and `Assembly.LoadFile` (#12203)
+- Allow case insensitive paths for determining `PSModulePath` (#12192)
+
+### General Cmdlet Updates and Fixes
+
+- Add `null` check for Windows PowerShell install path (#12296)
+- Fix Null Reference error in CSV commands (#12281) (Thanks @iSazonov!)
+- Fix `WinCompat` module loading to treat Core edition modules higher priority (#12269)
+- Fix `<img />` detection regex in web cmdlets (#12099) (Thanks @vex32!)
```
Comment (TravisEz13) [inline CHANGELOG/7.0.md:15]:
```suggestion
- Fix `<img />` detection regex in web cmdlets (#12099) (Thanks @vexx32!)
```

**Pull request merged**